LoopKit/DeviceManager/DeviceManager.swift

```diff
@@ -35,3 +35,8 @@ public protocol DeviceManager: CustomDebugStringConvertible, AlertResponder, Ale
     /// Is the device manager onboarded and ready for use?
     var isOnboarded: Bool { get }
 }
+
+extension DeviceManager {
+    public static var managerIdentifier: String { identifier }
```

Comment:
If we want to rename this, let's do it right, and actually rename it completely at call sites, and not have a variety of ways to access this.

Reply:
I think it should just be `identifier` and would just change to that. What do you think?

Reply:
That or pluginIdentifier.  `identifier` is so broad/vague, I worry a little about conflicts when applying it to a base protocol that a lot of different types will be required to conform to.